docs/developer/instruction/autoreduction.rst

```diff
@@ -0,0 +1,58 @@
+How to Modify an Instrument Autoreduction Configuration Page
+==============================
+
+.. note::
+    This document is a draft. If you use the instructions, please consider correcting/improving them.
+
+Instrument scientists can set up a form where Web Monitor users with staff status can modify the autoreduction script for the instrument.
+The web form lets the instrument scientists change parameter values in the autoreduction script and create a new version on the analysis cluster at
+
+``/<facility (SNS or HFIR)>/<instrument>/shared/autoreduce/reduce_<instrument>.py``
+
+from where it is fetched by the postprocessing agent during autoreduction. `Here <https://monitor.sns.gov/reduction/cncs/>`_ is an example of the form for the instrument CNCS
+(can only be viewed by logged in users with staff or superuser status).
+
+How to add a parameter to the configuration page
+-----------------------
+
+The following describes how an instrument scientist can add new parameters to an existing autoreduction configuration form.
+The instrument CNCS will be used as an example.
+
+1. On the analysis cluster, modify the template Python script for the instrument
+
+``/<facility (SNS or HFIR)>/<instrument>/shared/autoreduce/reduce_<instrument>.py.template``
+
+to define the new templated variable and the logic using it, e.g.::
```

Comment:
It would be good to mention that the `reduce_<instrument>.py.template` is filled in as a python formatting string with the keys/values provided in a `dict`